src/devices/Dhtxx/README.md

```diff
@@ -21,13 +21,29 @@ The DHT temperature and humidity sensors are very popular. This projects support
 // GPIO Pin
 using (Dht11 dht = new Dht11(26))
 {
-    Temperature temperature = dht.Temperature;
-    double humidity = dht.Humidity;
+    var temperature = dht.Temperature;
+    var humidity = dht.Humidity;
+    // You can only display temperature and humidity if the read is successful otherwise, this will raise an exception as
+    // both temperature and humidity are NAN
+    if (dht.IsLastReadSuccessful)
+    {
+        Console.WriteLine($"Temperature: {temperature.DegreesCelsius} \u00B0C, Humidity: {humidity.Percent} %");
+
+        // WeatherHelper supports more calculations, such as saturated vapor pressure, actual vapor pressure and absolute humidity.
+        Console.WriteLine(
+            $"Heat index: {WeatherHelper.CalculateHeatIndex(temperature, humidity).Celsius:0.#}\u00B0C");
+        Console.WriteLine(
+            $"Dew point: {WeatherHelper.CalculateDewPoint(temperature, humidity).Celsius:0.#}\u00B0C");
+    }
+    else
+    {
+        Console.WriteLine("Error reading DHT sensor");
+    }
 }
 ```
-**Note:** _Specifically on the RPi with the DHT22, 1-Wire works using Raspian but not with IoT-Core. The device has to switch the 1-wire pin between input and output and vice versa. It seems that Windows IoT-Core OS can't switch the pin direction quick enough. There have been suggestions for using two pins; one for input and one for output. This solution has not been implemented here, but these are some handy links that may help setting that up:_
+**Note:** _Specifically on the RPi with the DHT22, 1-Wire works using Raspian but not with Windows 10 IoT Core. The device has to switch the 1-wire pin between input and output and vice versa. It seems that Windows IoT Core OS can't switch the pin direction quick enough. There have been suggestions for using two pins; one for input and one for output. This solution has not been implemented here, but these are some handy links that may help setting that up:_
```

Comment:
I think this applies for the DHT11, too. This also uses one wire for communication and switches between input and output.

Reply:
Yes, correct. It actually applies to all of them. I'll aslo then add to the Q&A